src/views/adminSettings.pug

```diff
@@ -2,13 +2,24 @@ extends layout.pug
 
 block content
   h2= lang('ADMIN_SETTINGS_USERS_HEADER')
-  each user in users
-    span.is-size-6.inline= user.id
-    a(href=`${_CC.config.base}admin-settings/edit/${user.id}`)
-      span.is-size-7.icon.has-text-info
-        i.fas.fa-edit
-      span.is-sr-only
-        = lang('ADMIN_SETTINGS_USERS_EDIT')
+  table.table 
+    thead   
+      th=lang('ADMIN_SETTINGS_TABLE_USERNAME')
+      th=lang('ADMIN_SETTINGS_TABLE_ADMIN_USER')
+      th=lang('ADMIN_SETTINGS_TABLE_WISHLIST_COUNT')
+      th=lang('ADMIN_SETTINGS_TABLE_EDIT')
+    tbody 
+      each user in users
+        tr(id=user.id)
+          td=user.id
+          td=user.doc.admin 
```

Comment:
I'd like this to use the localized equivalent of yes or no rather than true/false. Something like:
```js
= user.doc.admin ? lang('YES') : lang('NO')
```

Other than that this looks great! It's a lot more intuitive, thank you.

Reply:
Yep, makes sense. Done in https://github.com/Wingysam/Christmas-Community/pull/149/commits/cc185ca7618899779e794f5b2b80092727f338f1

![image](https://github.com/user-attachments/assets/9af1b852-10aa-4125-93ad-7e6c47392c55)